test/ctia/stores/es/init_test.clj

```diff
@@ -87,6 +87,52 @@
       (is (= "2s" refresh_interval)))
     (index/delete! conn (str indexname "*"))))
 
+(deftest get-existing-indices-test
+  (index/delete! es-conn (str indexname "*"))
+  (let [exited? (atom false)
```

Comment:
Maybe I'd rename to `exited?` to `successful?`

That way the 'good ' tests say `true` instead of `false`, which seems more natural. Else it reads like some sort of double negative